### PR TITLE
WL-4079 Changing ical export information to have export period from config property

### PR DESCRIPTION
**Pull request by surya-janani**
Adding ical export information to publish ( private )

### CODE REVIEW

File: calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarAction.java [hunk 1]
```diff
@@ -4041,6 +4041,10 @@ public String buildIcalExportPanelContext(VelocityPortlet portlet, Context conte
 		boolean exportEnabled = CalendarService.getExportEnabled(calId);
 		context.put("enable_export", String.valueOf(exportEnabled) );
 
+		String icalInfoArr[] = {String.valueOf(ServerConfigurationService.getInt("calendar.export.previous.months",6)),String.valueOf(ServerConfigurationService.getInt("calendar.export.next.months",6)),rb.getFormattedMessage("ical.info.public")};
```
Comment (buckett) [inline calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarAction.java]: Building up strings like this is generally bad and it's better to just edit the template and get the addition string from the bundle.

Comment (buckett) [inline calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarAction.java]: Also can you wrap you're lines so they aren't as long. Sort of unwritten rule is over 120 characters is too much (although it's not hard and fast).

**Comment by surya-janani** [comment]
You mean just with the third parameter right?

File: calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarAction.java (hunk 1 above)
Comment (buckett) [inline calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarAction.java]: Yep, 3rd parameter.

**Comment by surya-janani** [comment]
Making review changes

File: calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarAction.java [hunk 2]
```diff
@@ -4041,6 +4041,11 @@ public String buildIcalExportPanelContext(VelocityPortlet portlet, Context conte
 		boolean exportEnabled = CalendarService.getExportEnabled(calId);
 		context.put("enable_export", String.valueOf(exportEnabled) );
 
+		String icalInfoArr[] = {String.valueOf(ServerConfigurationService.getInt("calendar.export.previous.months",6)),
+			String.valueOf(ServerConfigurationService.getInt("calendar.export.next.months",6))};
```
Comment (buckett) [inline calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarAction.java]: I think you have the arguments the wrong way round for the parameter substitution. You're bundle in plus/minus but your array is previous/next.

**Comment by surya-janani** [comment]
Review changes : Putting params in the right order

File: calendar-tool/tool/src/webapp/vm/calendar/chef_calendar_icalexport.vm [hunk 3]
```diff
@@ -25,7 +25,7 @@
 	</h3>
 	
 	<p class="messageInformation">
-		$tlang.getString("ical.info")<br/>
+		$icalInfoStr $tlang.getString("ical.info")<br/>
```
Comment (buckett) [inline calendar-tool/tool/src/webapp/vm/calendar/chef_calendar_icalexport.vm]: This doesn't look right, `ical.info` needs substituting when output, should it be `ical.info.public`?

**Comment by surya-janani** [comment]
Oops..Really sorry about overlooking that.  Made the changes

**Comment by buckett** [comment]
Could you squash this set of commits into one please.

**Comment by surya-janani** [comment]
Rebased

**Pull request merged**